CROS_RUST_REMOVE_DEV_DEPS didn't cover these unneeded dependencies.
diff --git a/Cargo.toml b/Cargo.toml
index 862a0a9..211eda9 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -54,9 +54,3 @@ version = "0.1.10"
 [dependencies.libc]
 version = "0.2.78"
 features = ["extra_traits"]
-[target."cfg(any(target_os = \"android\", target_os = \"linux\"))".dev-dependencies.caps]
-version = "0.3.1"
-[target."cfg(target_os = \"dragonfly\")".build-dependencies.cc]
-version = "1"
-[target."cfg(target_os = \"freebsd\")".dev-dependencies.sysctl]
-version = "0.1"
